builder/lxc/config.go

```diff
@@ -29,6 +30,18 @@ type Config struct {
 	ctx interpolate.Context
 }
 
+type ExportConfig struct {
+	Filename string  `mapstructure:"filename"`
```

Comment:
oh, got it. I though it will require the json property to have the first capital letter.